<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loading</class>
 <widget class="QDialog" name="loading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="loading_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>loading......</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
